build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         classpath 'gradle.plugin.net.minecrell:licenser:0.3'
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
         classpath 'gradle.plugin.org.spongepowered:spongegradle:0.8.1'
-        classpath 'gradle.plugin.org.spongepowered:event-impl-gen:5.0.2'
+        classpath 'gradle.plugin.org.spongepowered:event-impl-gen:5.4.0'
```

Comment:
5.5.1 ?

Reply:
Ya, didn't realize that. was developing railcraft in the last few hours so did not update this yet.